### DISCUSSION TITLE
[Suggestions] Should we indicate the terrain penalty for objects where the hero can move

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#6116] [General]
In fheroes2, should we indicate the terrain penalty for objects where the hero can move.

In fheroes2:

https://user-images.githubusercontent.com/43583994/200309009-351d1151-8316-4659-b811-83eb32be7613.mp4

Related discussion: https://github.com/ihhub/fheroes2/discussions/3291.

**Comment by zenseii** [comment]
Yep, I agree that this is a problem, especially for objects placed right where a terrain changer. However it might clutter up the UI to have the extra text on every object.
